dev-python/virtualenvwrapper/metadata.xml

```diff
@@ -5,8 +5,17 @@
 		<email>python@gentoo.org</email>
 		<name>Python</name>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>parona@protonmail.com</email>
+		<name>Alfred Wingate</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 	<upstream>
 		<remote-id type="pypi">virtualenvwrapper</remote-id>
 		<remote-id type="bitbucket">dhellmann/virtualenvwrapper</remote-id>
```

Comment:
Are they still using bitbucket or can we drop it?

Reply:
The bitbucket is entirely inactive, I just left in so that pkgcheck would have less noise about from -r2 HOMEPAGE.

If pkgcheck noise is the lesser sin the dropping it would be the best option.

Reply:
Ah, fair enough. Let's drop it on cleanup then.